### PR TITLE
Some cargo and prolathe additions and edits.

### PR DESCRIPTION
**Pull request by thingy404**
>Stocked some more things to crates that were too empty, fixed crates that came without the equal ammount of sister items. (3 armor vests, 2 helmets.)

>Added a taser shotgun ammo crate.

>Implemented the commented out crates for hospitality as vendor refills for the sake of filling it up.

>Removed one metal stack from bulk metal crate, replaced with rods for commodity until craft recipes add the option to craft multiple rods. (MIGHT BE BUGGY. if it spawns too much metal, remove the "Ammount" line.)

>Added a new blood pack crate for 0- only, way more expensive than the variety crate.

>BEES, BEES, BEES! (beekeping crate is now implemented)

>There was amining explosives crate. I unlocked it. 5 charges for 1500 sounds fair.

>Added shit to the mining supply crate such as a shovel, pickaxe, etc. increased price for it too.

>Janitorial crates get two more cleaner bottles, because it's always crew who can't find the bottles that order these.

>Random guns crates now have 3 guns, higher chance for cheap gun

>Random costumes have 5 picks now, cause why only 2?!??!?

>Revamped the contraband crate, now named "Black market crate"

>Added a new disk called Postal Pack that includes every gun in every disk. made every ammo disk unlimited uses.

>Added EXPENSIVE crates with the good unlimited uses disks that are not ammo.

>Added a nice lil condiment crate with ketchup, mayonnaise, mustard... just pick chef already!

>And a few nice new crates for hospitality. Finally!

### CODE REVIEW

File: code/modules/cargo/packs.dm [hunk 1]
```diff
@@ -324,6 +390,39 @@ var/list/all_supply_groups = list("Operations","Security","Hospitality","Enginee
...
+	containertype = /obj/structure/closet/crate/freezer
+	crate_name = "Mistery soup crate"
+	group = "Hospitality"
+
+/datum/supply_pack/mistery_soup_deluxe_crate
+	name = "Mystery soup deluxe crate"
+	contains = list(/obj/item/weapon/reagent_containers/food/snacks/wishsoup)
+	cost = 1200
+	containertype = /obj/structure/closet/crate/freezer
+	crate_name = "Mistery soup deluxe crate"
```
Comment (improvedname) [inline code/modules/cargo/packs.dm]: You spelled mystery wrong

File: code/modules/cargo/packs.dm [hunk 2]
```diff
@@ -324,6 +390,39 @@ var/list/all_supply_groups = list("Operations","Security","Hospitality","Enginee
...
+					/obj/item/weapon/reagent_containers/food/snacks/mysterysoup,
+					/obj/item/weapon/reagent_containers/food/snacks/mysterysoup,)
+	cost = 1000
+	containertype = /obj/structure/closet/crate/freezer
+	crate_name = "Mistery soup crate"
+	group = "Hospitality"
+
+/datum/supply_pack/mistery_soup_deluxe_crate
+	name = "Mystery soup deluxe crate"
+	contains = list(/obj/item/weapon/reagent_containers/food/snacks/wishsoup)
```
Comment (improvedname) [inline code/modules/cargo/packs.dm]: This is litteraly boiled water for 1200 credits

File: code/modules/cargo/packs.dm [hunk 3]
```diff
@@ -1152,6 +1287,9 @@ var/list/all_supply_groups = list("Operations","Security","Hospitality","Enginee
 					/obj/item/weapon/storage/bag/trash,
 					/obj/item/device/lightreplacer,
 					/obj/item/weapon/reagent_containers/spray/cleaner,
+					/obj/item/weapon/reagent_containers/spray/cleaner,
+					/obj/item/weapon/reagent_containers/spray/cleaner,
+					/obj/item/weapon/reagent_containers/spray/cleaner,
```
Comment (improvedname) [inline code/modules/cargo/packs.dm]: Too much space cleaner.

Comment (improvedname) [reply code/modules/cargo/packs.dm]: Chemistry can give him refills

File: code/game/objects/items/weapons/autolathe_disks.dm [hunk 4]
```diff
@@ -398,6 +398,10 @@
 		/datum/design/autolathe/gun/sniperrifle
 	)
 
+/obj/item/weapon/computer_hardware/hard_drive/portable/design/nt_old_guns/priest
+	disk_name = "Neotheology Factory of the Old Testament"
+	license = -1
```
Comment (improvedname) [inline code/game/objects/items/weapons/autolathe_disks.dm]: no

File: code/game/objects/items/weapons/autolathe_disks.dm [hunk 5]
```diff
@@ -412,6 +416,9 @@
 		/datum/design/autolathe/gun/ionrifle,
 	)
 
+/obj/item/weapon/computer_hardware/hard_drive/portable/design/nt_new_guns/priest
+	disk_name = "Neotheology Factory of the New Testament"
+	license = -1
```
Comment (improvedname) [inline code/game/objects/items/weapons/autolathe_disks.dm]: no

File: code/game/objects/items/weapons/autolathe_disks.dm [hunk 6]
```diff
@@ -449,6 +456,10 @@
 		/datum/design/autolathe/gun/pump_shotgun,
 	)
 
+/obj/item/weapon/computer_hardware/hard_drive/portable/design/fs_cheap_guns/factory
+	disk_name = "Frozen Star Basic Civilian Factory Pack"
+	license = -1
```
Comment (improvedname) [inline code/game/objects/items/weapons/autolathe_disks.dm]: no

File: code/game/objects/items/weapons/autolathe_disks.dm [hunk 7]
```diff
@@ -474,6 +485,10 @@
 		/datum/design/autolathe/gun/grenade_launcher_lenar, //GLs
 	)
 
+/obj/item/weapon/computer_hardware/hard_drive/portable/design/fs_kinetic_guns/factory
+	disk_name = "Frozen Star Ultimate Protection Factory Pack"
+	license = -1
```
Comment (improvedname) [inline code/game/objects/items/weapons/autolathe_disks.dm]: no

File: code/game/objects/items/weapons/autolathe_disks.dm [hunk 8]
```diff
@@ -485,11 +500,52 @@
...
+		/datum/design/autolathe/clothing/excelsior_helmet
+	)
+
+
 /obj/item/weapon/computer_hardware/hard_drive/portable/design/nonlethal_ammo //please, maintain general order (pistol>speedloaders>smg>other>shells)+(smaller/less damaging caliber>bigger/more damaging caliber)
 	disk_name = "Frozen Star Nonlethal Ammo Pack"
 	icon_state = "frozenstar"
 
-	license = 20
+	license = -1
```
Comment (improvedname) [inline code/game/objects/items/weapons/autolathe_disks.dm]: no

File: code/game/objects/items/weapons/autolathe_disks.dm [hunk 9]
```diff
@@ -514,7 +570,7 @@
 	disk_name = "Frozen Star Lethal Ammo Pack"
 	icon_state = "frozenstar"
 
-	license = 20
+	license = -1
```
Comment (improvedname) [inline code/game/objects/items/weapons/autolathe_disks.dm]: all the same as above, dont make them unlimited.

File: code/game/objects/items/weapons/autolathe_disks.dm [hunk 10]
```diff
@@ -485,11 +500,52 @@
 		/datum/design/autolathe/gun/pulse_rifle_cassad,
 	)
 
+/obj/item/weapon/computer_hardware/hard_drive/portable/design/P_apocalypse_guns
+	disk_name = "Postal Apocalypse Pack"
+	desc = "Perfect for those Postal days."
+	icon_state = "neotheology"
```
Comment (improvedname) [inline code/game/objects/items/weapons/autolathe_disks.dm]: Whats this pack for, it doesn't make sense exhelsior is a criminal organization and this voids the point of surplus weapons

File: code/modules/cargo/packs.dm [hunk 11]
```diff
@@ -772,7 +872,7 @@ var/list/all_supply_groups = list("Operations","Security","Hospitality","Enginee
 
 /datum/supply_pack/medical
 	name = "Medical crate"
-	contains = list(/obj/item/weapon/storage/firstaid/regular,
+	contains = list(/obj/item/weapon/storage/firstaid/adv,
```
Comment (improvedname) [inline code/modules/cargo/packs.dm]: keep regular in it and increase price by 500 credits

File: code/modules/cargo/packs.dm [hunk 12]
```diff
@@ -807,6 +907,14 @@ var/list/all_supply_groups = list("Operations","Security","Hospitality","Enginee
 	crate_name = "Coolant tank crate"
 	group = "Medical / Science"
 
+/datum/supply_pack/death_implant_crate
+	name = "Death implant Crate"
+	contains = list(/obj/item/weapon/storage/box/cdeathalarm_kit)
+	cost = 900
```
Comment (improvedname) [inline code/modules/cargo/packs.dm]: make this 1000

File: code/modules/cargo/packs.dm [hunk 13]
```diff
@@ -1086,8 +1205,15 @@ var/list/all_supply_groups = list("Operations","Security","Hospitality","Enginee
...
 					/obj/item/device/lighting/toggleable/flashlight,
-					/obj/item/weapon/tool/pickaxe/jackhammer)
-	cost = 1000
+					/obj/item/weapon/tool/pickaxe/jackhammer,
+					/obj/item/weapon/tool/pickaxe,
+					/obj/item/weapon/tool/shovel,
+					/obj/item/weapon/storage/bag/ore,
+					/obj/item/device/scanner/analyzer,
+					/obj/item/clothing/glasses/powered/meson,
+					/obj/item/device/radio/headset/headset_cargo,
```
Comment (improvedname) [inline code/modules/cargo/packs.dm]: Not needed

File: code/modules/cargo/packs.dm [hunk 14]
```diff
@@ -1336,7 +1490,24 @@ var/list/all_supply_groups = list("Operations","Security","Hospitality","Enginee
...
+				/obj/item/weapon/reagent_containers/food/snacks/candy,
+				/obj/item/weapon/reagent_containers/food/snacks/candy_corn,
+				/obj/item/weapon/reagent_containers/food/snacks/fishfingers
+				/obj/item/weapon/reagent_containers/food/snacks/cookie,
+				/obj/item/weapon/reagent_containers/food/snacks/chocolateegg,
+				/obj/item/weapon/reagent_containers/food/snacks/donut/chaos,
+				/obj/item/weapon/reagent_containers/food/snacks/bearmeat,
+				/obj/item/weapon/reagent_containers/food/snacks/cheeseburger,
+				/obj/item/weapon/reagent_containers/food/snacks/fishburger,
+				/obj/item/weapon/reagent_containers/food/snacks/muffin,)
```
Comment (improvedname) [inline code/modules/cargo/packs.dm]: theres a , at the end it'll cause a conflict error

File: code/modules/cargo/packs.dm [hunk 15]
```diff
@@ -1245,15 +1383,31 @@ var/list/all_supply_groups = list("Operations","Security","Hospitality","Enginee
...
+					/datum/design/autolathe/gun/pump_shotgun,
+					/datum/design/autolathe/gun/revolver_detective,
+					/datum/design/autolathe/gun/deagle,
+					/datum/design/autolathe/gun/ak47_fs,
+					/obj/item/weapon/reagent_containers/glass/bottle/cyanide,
+					/obj/item/weapon/reagent_containers/glass/bottle/chloralhydrate,
+					/obj/item/weapon/reagent_containers/pill/methylphenidate,
+					/obj/item/weapon/reagent_containers/pill/floorpill,
+					/obj/item/weapon/storage/box/explosive,
+					/obj/item/weapon/storage/box/anti_photons,)
```
Comment (improvedname) [inline code/modules/cargo/packs.dm]: Conflict error

File: code/modules/cargo/packs.dm [hunk 16]
```diff
@@ -324,6 +390,39 @@ var/list/all_supply_groups = list("Operations","Security","Hospitality","Enginee
...
+	crate_name = "fortune cookie crate"
+	group = "Hospitality"
+
+/datum/supply_pack/mistery_soup_crate
+	name = "Mystery soup crate"
+	contains = list(/obj/item/weapon/reagent_containers/food/snacks/mysterysoup,
+					/obj/item/weapon/reagent_containers/food/snacks/mysterysoup,
+					/obj/item/weapon/reagent_containers/food/snacks/mysterysoup,
+					/obj/item/weapon/reagent_containers/food/snacks/mysterysoup,
+					/obj/item/weapon/reagent_containers/food/snacks/mysterysoup,)
```
Comment (improvedname) [inline code/modules/cargo/packs.dm]: Conflict error

File: code/modules/cargo/packs.dm [hunk 17]
```diff
@@ -324,6 +390,39 @@ var/list/all_supply_groups = list("Operations","Security","Hospitality","Enginee
 	crate_name = "bartending supply crate"
 	group = "Hospitality"
 
+/datum/supply_pack/fortune_cookie_crate
+	name = "Fortune cookie crate"
+	contains = list(/obj/item/weapon/reagent_containers/food/snacks/fortunecookie,
+					/obj/item/weapon/reagent_containers/food/snacks/fortunecookie,
+					/obj/item/weapon/reagent_containers/food/snacks/fortunecookie,
+					/obj/item/weapon/reagent_containers/food/snacks/fortunecookie,
+					/obj/item/weapon/reagent_containers/food/snacks/fortunecookie,)
```
Comment (improvedname) [inline code/modules/cargo/packs.dm]: Conflict error

File: code/modules/cargo/packs.dm [hunk 18]
```diff
@@ -155,7 +206,9 @@ var/list/all_supply_groups = list("Operations","Security","Hospitality","Enginee
 	name = "FS Assault Pack"
 	contains = list(/obj/item/weapon/gun/projectile/automatic/ak47/fs,
 			/obj/item/weapon/gun/projectile/automatic/idaho,
-			/obj/item/weapon/gun/projectile/automatic/idaho)
+			/obj/item/weapon/gun/projectile/automatic/idaho,
+			/obj/item/weapon/gun/projectile/automatic/wt550
```
Comment (Trilbyspaceclone) [inline code/modules/cargo/packs.dm]: Needs a , at the end

File: code/modules/cargo/packs.dm [hunk 19]
```diff
@@ -1030,41 +1154,36 @@ var/list/all_supply_groups = list("Operations","Security","Hospitality","Enginee
 	containertype = /obj/structure/largecrate
 	crate_name = "Water tank crate"
 	group = "Hydroponics"
-/*
+
 /datum/supply_pack/bee_keeper
```
Comment (Trilbyspaceclone) [inline code/modules/cargo/packs.dm]: I think this should stay locked/contraband
as well as not in game

Comment (thingy404) [reply code/modules/cargo/packs.dm]: it's literally bee keeping shit. let the gardeners have their sugary insect vomit.

File: code/game/objects/items/weapons/autolathe_disks.dm (hunk 10 above)
Comment (thingy404) [reply code/game/objects/items/weapons/autolathe_disks.dm]: contraband?

File: code/modules/cargo/packs.dm (hunk 13 above)
Comment (thingy404) [reply code/modules/cargo/packs.dm]: just a basic miner gear locker.

File: code/modules/cargo/packs.dm (hunk 2 above)
Comment (king5327) [reply code/modules/cargo/packs.dm]: 👍

**Comment by king5327** [comment]
I'm going to start by mentioning that factory packs are supposed to be unique, unlimited (but not copyable) disks that are either adminspawn, or, in rare cases, given to the head of staff.

**Comment by DogZeroX** [comment]
Fix your PR. REEEE.

File: code/game/objects/items/weapons/autolathe_disks.dm [hunk 20]
```diff
@@ -580,7 +636,7 @@
 
 /obj/item/weapon/computer_hardware/hard_drive/portable/design/excelsior_weapons
 	disk_name = "Excelsior Means of Revolution"
-	desc = "The back has a machine etching: \"We stand for organized terror - this should be frankly admitted. Terror is an absolute necessity during times of revolution.\""
+	desc = "The back has a machine etching: \"We stand for organized terror - this should be frankly admitted. Terror is an absolute necessity during times of revolution."
```
Comment (Trilbyspaceclone) [inline code/game/objects/items/weapons/autolathe_disks.dm]: add back in the \ as its to got with the machine etching

File: code/modules/cargo/packs.dm [hunk 21]
```diff
@@ -111,6 +111,54 @@ var/list/all_supply_groups = list("Operations","Security","Hospitality","Enginee
...
+	name = "Postal Pack Disk"
+	contains = list(/obj/item/weapon/computer_hardware/hard_drive/portable/design/P_apocalypse_guns)
+	cost = 8000
+	crate_name = "Disk Crate"
+	contraband = TRUE
+	group = "Operations"
+
+/datum/supply_pack/fs_warrior_factory_disk
+	name = "FS Void Warrior Factory Disk"
+	contains = list(/obj/item/weapon/computer_hardware/hard_drive/portable/design/fs_kinetic_guns/factory)
```
Comment (Trilbyspaceclone) [inline code/modules/cargo/packs.dm]: Wrong one
Also lets not give cargo endless gun disks

File: code/modules/cargo/packs.dm [hunk 22]
```diff
@@ -111,6 +111,54 @@ var/list/all_supply_groups = list("Operations","Security","Hospitality","Enginee
 	crate_name = "Export scanners crate"
 	group = "Operations"
 
+/datum/supply_pack/postal_disk
+	name = "Postal Pack Disk"
+	contains = list(/obj/item/weapon/computer_hardware/hard_drive/portable/design/P_apocalypse_guns)
+	cost = 8000
```
Comment (Trilbyspaceclone) [inline code/modules/cargo/packs.dm]: you forgot to add a few more 0's to that

File: code/modules/cargo/packs.dm [hunk 23]
```diff
@@ -1245,15 +1380,31 @@ var/list/all_supply_groups = list("Operations","Security","Hospitality","Enginee
...
-					/obj/item/weapon/contraband/poster,
-					/obj/item/weapon/reagent_containers/food/drinks/bottle/pwine)
-	name = "Contraband crate"
+					/obj/item/weapon/rig/ihs_combat/equipped,
+					/obj/item/weapon/reagent_containers/food/drinks/bottle/pwine,
+					/obj/item/clothing/gloves/stungloves,
+					/obj/item/weapon/storage/belt/security,
+					/obj/item/weapon/mining_charge,
+					/datum/design/autolathe/tool/chainsaw,
+					/datum/design/autolathe/gun/large_energy_crossbow,
```
Comment (Trilbyspaceclone) [inline code/modules/cargo/packs.dm]: This and the deagle should not be in this crate
The stungloves are also pushing it to

**Comment by APWill** [comment]
Closed on Thingy 404's request.

**Pull request closed without merging**